src/components/Footer/Footer.module.scss

```diff
@@ -41,6 +41,16 @@
     font-size: rem(1.6) !important;
     color: var(--core-color-solid-slate-1400, #000);
     text-align: left;
+  }
+
+  .SectionTitle2 {
+    font-family: 'Ubuntu', sans-serif;
+    font-size: rem(1.6) !important;
+    color: var(--core-color-solid-slate-1400, #000);
+    text-align: left;
+  }
```

Comment:
Do we need this `SectionTitle2`. isn't the `SectionTitle` is having the same style?